agent/php_rinit.c

```diff
@@ -91,6 +91,16 @@ PHP_RINIT_FUNCTION(newrelic) {
       "(^([a-z_-]+[_-])([0-9a-f_.]+[0-9][0-9a-f.]+)(_{0,1}.*)$|(.*))",
       NR_REGEX_CASELESS, 0);
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
+     && !defined OVERWRITE_ZEND_EXECUTE_DATA
+  /*
+   * Pre-OAPI, this variables were kept on the call stack and
+   * therefore had no need to be in an nr_stack
+   */
+  nr_stack_init(&NRPRG(wordpress_tags), 32);
```

Comment:
Is there a reason the value `32` was chosen?

Reply:
I accidentally chose the `NR_STACK_DEFAULT_CAPACITY` value instead of using the macro. fixed in [de23c51](https://github.com/newrelic/newrelic-php-agent/pull/559/commits/de23c51e5e08e3661282d595b47809ec91d25ded)